packages/tools/client-debugger/client-debugger/src/Logs.ts

```diff
@@ -44,6 +44,22 @@ export interface ConnectionStateChangeLogEntry extends StateChangeLogEntry<Conne
 	 * Will always be undefined for disconnects.
 	 */
 	clientId: string | undefined;
+
+	/**
+	 * The ID of container that has state changed.
+	 */
+	containerId: string | undefined;
```

Comment:
Do we need to track the ContainerID on these? A given debugger will always be associated with exactly 1 Container, so this should never change.

Reply:
Right right. Our debugger is 1 to 1 map to container. Remove it from here.